Portal/View/RefreshControl.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  RefreshControl.swift
+//  Portal
+//
+//  Created by Cristian Ames on 9/5/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import UIKit
+
+class RefreshController<MessageType> {
```

Comment:
~~This should not be here because is `UIKit` dependant. It should be inside the renderers folders in a file called `RefreshControlRenderer`~~

Reply:
`RefreshController` should not be necessary, check my comments bellow in `PortalCollectionView`